dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,35 @@
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
+	docker-compose down
+.PHONY: docker-down
+
+docker-start-dev:
```

Comment:
And some shells have makefile autocomplete, so it's nice to have a dedicated command IMO